Source/Mosa.Kernel.BareMetal/Platform.cs

```diff
@@ -24,6 +24,8 @@ public static void ConsoleWrite(byte c)
 	public static void DebugWrite(byte c)
 	{ }
 
+	public static Firmware GetFirmware() => default;
```

Comment:
What is firmware?

Reply:
Firmware is a struct containing information about the firmware, such as the vendor or the version. It's similar to the CPU and Time structs.